versioned_docs/version-0.2.0/glossary.mdx

```diff
@@ -0,0 +1,11 @@
+## Glossary
+
+1. Slice Boundaries: The time required for processing depends upon the amount of data. When the amount of data
+to be replicated is massive, its efficient to seperate the data into slices and process it seperately.
+Slice boundaries are basically resources, with instructions that maintain the slices and also manages communication
+of the pods and services across clusters, edges, clouds, data centers etc.
+
+2. RFC1918 network addresses: The rapid growth of internet led to the exhaustion of the available ipV4 addresses, as a solution subnets 
+were introduced that an organisation could use internally. RFC1918 are requests for comment that defines a set of private
+ipv4 addresses that are not routeable on the internet. Any router connected to the internet will discard packets of data
+defined on the RFC1918.
```

Comment:
Either it should be `... in the RFC1918 record`. or `.., in RFC1918`. 

I have always heard of referring RFC as specifications but for RFC1918, I could only find it to be referred to as a record.

cc @pdorbala